docs/overview/configuration.md

```diff
@@ -70,6 +70,44 @@ example.com
 *example.com
 ```
 
+### Intercepted Resources
+
+This allows you to intercept resources and modify them before they are loaded. This is useful for modifying the response of a request.
+
+```js
+const hero = new Hero({
+  interceptedResources: [
+    {
+      url: 'https://example.com',
+      body: '<div>Hello World!</div>',
+    },
+  ],
+});
+
+const heroStatus = new Hero({
+  interceptedResources: [
+    {
+      url: 'https://example.com',
+      statusCode: 404,
```

Comment:
Does this work? Is the system smart enough to send an empty body back?

Reply:
Why is this a requirement? 404 statuscodes don't force an empty body. If a user wants to make sure the body is empty they can set body and statusCode.

Reply:
For which statuscodes? Only 404 or others aswell

Reply:
Any response code. I think as long as you call end(), it will be fine